data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -162,6 +162,9 @@ then
 		--volume "$PWD:/assets:z" \
 		"${CONFIG_OPERATOR_IMAGE}" \
 		/usr/bin/cluster-config-operator render \
+		--cloud-provider-config-input-file=/assets/manifests/cloud-provider-config.yaml \
```

Comment:
maybe what we can do is add a condtion write before the run of this operator like here https://github.com/openshift/installer/pull/3831/files#diff-ce82c1d8a44f7dfc41dfc024085ccfeeR160
```
INPUT_CLOUD_CONFIG_FILE=""
if [ -f "/opt/manifests/cloud-provider-config.yaml" ]; then
     INPUT_CLOUD_CONFIG_FILE="--cloud-provider-config-input-file=/assets/manifests/cloud-provider-config.yaml"
fi
```
```suggestion
		"${ADDITIONAL_FLAGS}" \
```

Reply:
This way we don't have to change the cluster-config-operator to handle if not exists file case..
as we already allow input flag to be empty..